__tests__/fungible-token.spec.ts

```diff
@@ -0,0 +1,287 @@
+import path from 'path';
+import {Buffer} from 'buffer';
+import {Runner, BN, Account} from '..';
+
+const STORAGE_BYTE_COST = '10000000000000000000';
+
+async function init_ft(
+  ft: Account,
+  owner: Account,
+  supply: BN | string = '10000',
+) {
+  await ft.call(ft, 'new_default_meta', {
+    owner_id: owner,
+    total_supply: supply,
+  });
+}
+
+async function init_defi(defi: Account, ft: Account) {
+  await defi.call(defi, 'new', {
+    fungible_token_account_id: ft,
+  });
+}
+
+async function registerUser(ft: Account, user: Account) {
+  await user.call(
+    ft,
+    'storage_deposit',
+    {account_id: user},
+    // Deposit pulled from ported sim test
+    {attachedDeposit: new BN(STORAGE_BYTE_COST).mul(new BN(125))},
+  );
+}
+
+describe(`Running on ${Runner.getNetworkFromEnv()}`, () => {
+  let runner: Runner;
+  jest.setTimeout(60_000);
+
+  beforeAll(async () => {
+    runner = await Runner.create(async ({root}) => ({
+      ft: await root.createAndDeploy(
+        'fungible-token',
+        path.join(__dirname, 'build', 'debug', 'fungible_token.wasm'),
+      ),
+      defi: await root.createAndDeploy(
+        'defi',
+        path.join(__dirname, 'build', 'debug', 'defi.wasm'),
+      ),
+      ali: await root.createAccount('ali'),
+    }));
```

Comment:
Because the tests have a different setup and the only common pattern was to initialize fungible token contract, but I liked having this within the test so that it's easy to see how many tokens are being initialized.

I'd rather not change this myself, as I really prefer having tests readable and have minimal side effects, but if you feel strongly about it feel free to change after.

I've noticed a similar pattern in the sim tests I ported and I really didn't like it